### PR TITLE
add --pro and --free options to support RTD Pro

### PR DESCRIPTION
**Pull request by VanDavv**
Hi @honzajavorek,

Thanks for a very helpful tool! I've tweaked it a little to support also the https://readthedocs.com/ page (not only .org)

### CODE REVIEW

**Comment by honzajavorek** [comment]
@VanDavv Hi, thanks for the PR! I see Travis is broken, but I like your changes, I think it makes sense. I suspect the CI is just outdated. It might take a while, but give me some time to look at this and I'm sure I'll accept your code soon 👍

**Comment by VanDavv** [comment]
@honzajavorek Great, thanks for the update!

File: rtd_redirects.py [hunk 1]
```diff
@@ -3,32 +3,34 @@
...
 
-    url = f'https://readthedocs.org/dashboard/{project_name}/redirects/'
+    url = f'{rtd_url}/dashboard/{project_name}/redirects/'
     res = session.get(url)
     res.raise_for_status()
     html_tree = html.fromstring(res.content)
     html_tree.make_links_absolute(res.url)
 
-    redirects = yaml.load(redirects_filename.read()).get('redirects')
+    redirects = yaml.safe_load(redirects_filename.read()).get('redirects')
```
Comment (honzajavorek) [inline rtd_redirects.py:27]: 👍 good catch, thanks!

**Review by honzajavorek — APPROVED** [review]
Hi again! Finally found time to go through this. Thanks a lot, I like the change 👍  I fixed the CI meanwhile, so could you please:

- [x] rebase on top of the current main branch
- [ ] add mention about this new feature in the README?

Then I'm happy to merge this and release a new version. Thanks!

**Comment by VanDavv** [comment]
@honzajavorek It's all done, great to see these quick fixes and engagement from your side, keep up the great work and thanks for the kind words :)

**Comment by honzajavorek** [comment]
Amazing, thanks for spending time and effort to make this tool better!

**Pull request merged**